src/rules/arrowParensRule.ts

```diff
@@ -20,21 +20,29 @@ import * as ts from "typescript";
 import * as Lint from "../index";
 import { hasModifier } from "../language/utils";
 
+const AVOID_ON_SINGLE_PARAMETER = "avoid-on-single-parameter";
```

Comment:
this should've used the rule option name I suggested here, "ban-single-arg-parens" https://github.com/palantir/tslint/issues/1860#issuecomment-267388635. We need to move towards a more predictable and consistent naming strategy for rules an options; this is diverging in the wrong direction. We don't use the token "avoid" anywhere else. See https://github.com/palantir/tslint/issues/1371